config/initializers/country_config.rb

```diff
@@ -23,7 +23,7 @@ class CountryConfig
       states: COUNTRYWISE_STATES["Bangladesh"],
       dashboard_locale: ENV["DEFAULT_PREFERRED_DASHBOARD_LOCALE"] || "en_BD",
       time_zone: ENV["DEFAULT_TIME_ZONE"] || "Asia/Dhaka",
-      faker_locale: "en-IND",
+      faker_locale: "en",
```

Comment:
Shouldn't the faker_locale be the same as dashboard_locale?

Reply:
Tests use faker_locale to populate sync data. If we were to keep it same as dashboard locale, tests will fail as they cannot find this data.